Source/AssistantV1/Models/RuntimeResponseGeneric.swift

```diff
@@ -1,5 +1,5 @@
 /**
- * (C) Copyright IBM Corp. 2018, 2019.
+ * (C) Copyright IBM Corp. 2019.
```

Comment:
Are these copyrights being generated correctly? Not showstopper for major release but we might want to look into this.